apps/backend/src/logger/entities/log.entity.ts

```diff
@@ -0,0 +1,24 @@
+import {
```

Comment:
to be removed, will include when we find a use case.

Reply:
done :)